content/release-10-15-0/whatsnew-10-15-0.md

```diff
@@ -34,18 +34,21 @@ For detailed information, see the [Cumulocity IoT DTM guide](/guides/dtm/#overvi
 
 For information on installation and operations, see the *Digital Twin Manager -  Installation & operations guide*.
 
-### Multiple LNS connections for Sigfox and Actility
+### Device Management
 
-The Sigfox and Actility agents now support the configuration of multiple connections per tenant. One connection can be associated with multiple devices. Each connection can be updated and deleted.
+### Extensible device registration
 
-![All devices](/images/release-notes/multiple_lns_connection.png)
+Extensible device registration is a new feature that allows microservice developers to implement their own device registration process without explicitly extending a single line of UI source code.
 
-For details, refer to [Sigfox](https://cumulocity.com/guides/protocol-integration/sigfox) or [Actility LoRa](https://cumulocity.com/guides/protocol-integration/lora-actility) in the *Protocol integration guide*.
+A custom input mask can be created using JSON schema and some specific endpoints that the microservice needs to provide. This is then finally displayed in a dropdown on the device registration page as soon as the microservice is subscribed to the tenant.
+
+Our revised device registration for LWM2M also uses this feature.
 
+For detailed information, see the [Cumulocity IoT EDR guide](/guides/dtm/#overview).
```

Comment:
Just a placeholder for now.
We need to get the EDR guide done: https://cumulocity.atlassian.net/browse/DM-1261

Reply:
This will probably not be a guide but a section in the Device Management section of the User guide.